alpha/alpha-ui/src/main/resources/static/scss/_dropdowns.scss

```diff
@@ -0,0 +1,21 @@
+// Custom Dropdown Styling
```

Comment:
License header. If these file is from other place we need to specify the source.

Reply:
> License header. If these file is from other place we need to specify the source.

These files are already included in webjars bootstrap-4.3.1.jar, I will remove the scss file from resources/static